packages/api-react/src/services/wallet.ts

```diff
@@ -2025,15 +2025,15 @@ export const walletApi = apiWithTag.injectEndpoints({
       {
         walletId: number;
         nftLauncherId: string;
-        nftCoinId: string;
+        nftCoinIds: string[];
```

Comment:
Please see my comment below. We should add another API here: `setNFTDIDBulk` that will call `setNftDidBulk()`